<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>131</width>
      <height>46</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Database Test Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Structure">
     <property name="text">
      <string>DB Dialog Test</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBoxUser">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelUser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Benutzer/utente</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxBed">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelBed">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bett/letto</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>931</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mittag/mezzogiorno</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Abend/sera</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxMenuMid">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>80</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxMenuEv">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>120</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string> bestellt/ord.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>bestellt/ord.</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewPatientDietary_req">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>100</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Attribute/aggettivo:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Weiter/continua</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zur√ºck/ritorna</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Supplier">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Supplier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPatientDietary_req_newRecord">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>New Record</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDate">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelBed_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dat.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveOrder">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_admin">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport_2"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionAccount"/>
    <addaction name="actionPassword"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAdmin"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="actionPassword">
   <property name="text">
    <string>Password</string>
   </property>
  </action>
  <action name="actionExport_2">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
